src/privileged/vpnRecommender/content/doorhanger/doorhanger.css

```diff
@@ -0,0 +1,203 @@
+/*! This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  color: #333;
+  background-color: #fff;
+  --theme-color: #333;
+  --theme-background-color: #fff;
+  --theme-hover-background-color: rgba(12, 12, 13, 0.1);
+  font-family: -apple-system, "Segoe UI", "Ubuntu", "Roboto", "Oxygen", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+  width: 280px;
+  padding: 0;
+  margin: 0;
+}
+
+body.dark {
+  --theme-color: #fff;
+  --theme-background-color: #4a4a4f;
+  --theme-hover-background-color: rgba(12, 12, 13, 0.5);
+}
+
+#picture {
+  width: 100%;
+  text-align: center;
+  height: 130px;
+  background: url("../images/Doorhanger Hero@1x.png");
+  background-size: contain;
+}
+
+@media (min--moz-device-pixel-ratio: 1.5),
```

Comment:
nit: you don't need the `min--moz-device-pixel-ratio`

Reply:
changed to "min-device-pixel-ratio".